### PR TITLE
Disable pinch-to-zoom in Installer

### PR DESCRIPTION
**Pull request by ahmedalsudani**
@fjvallarino @tejohnso this fixes https://github.com/Rise-Vision/rise-player-electron/issues/14#issuecomment-194393752. It only covers the installer, not presentations running in Chrome.

### CODE REVIEW

**Comment by fjvallarino** [comment]
:+1:

File: installer/ui/ipc.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 const ipc = require("electron").ipcRenderer;
+const webFrame = require("web-frame");
```
Comment (tejohnso) [inline installer/ui/ipc.js]: The preferred format for this is now require('electron').webFrame as in ipcRenderer above.

**Comment by ahmedalsudani** [comment]
@tejohnso updated

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**